packages/components/src/components/entities/actions.ts

```diff
@@ -152,10 +152,12 @@ function extractEntityTypeOptionFromRow(row: Map<string, any>): IEntityTypeOptio
     }
 }
 
-function getChosenParentData(model: EntityIdCreationModel, parentSchemaQueries: Map<string, SchemaQuery>, allowParents: boolean) : Promise<Partial<EntityIdCreationModel>> {
+function getChosenParentData(model: EntityIdCreationModel, parentSchemaQueries: Map<string, EntityDataType>, allowParents: boolean) : Promise<Partial<EntityIdCreationModel>> {
```

Comment:
Maybe rename the parameter to `parentDataTypes` instead of `parentSchemaQueries`.

Reply:
Done.